Add `latest` as a symlink to `master` dir

mbrukman opened the pull request.

robertdale commented:
- Updated groovy link

robertdale commented:
Just realized that this is generated content.  Fixed in JanusGraph/janusgraph#46

mbrukman commented:
@robertdale — that's correct, the HTML files are generated from the text files in the main repo. Sorry for the confusion.

This should be documented; if you'd like to submit a PR to start a README.md file with some basic content including this information, that would be appreciated!

pluradj commented:
updating the repo description would help too @mbrukman

mbrukman commented:
@pluradj — I updated repo description to:

> JanusGraph documentation (generated content)

Thoughts?

pluradj commented:
Add a pointer to the docs source code location, then I think it's all good https://github.com/JanusGraph/janusgraph/tree/master/docs

traviscibot committed:
Deploy janusgraph/docs.janusgraph.org to github.com/janusgraph/docs.janusgraph.org.git:master

mbrukman committed:
Create CNAME to point to docs.janusgraph.org

The site https://docs.janusgraph.org is currently returning 404
due to missing this file, which was not brought over from the
previous version of this repo.

Signed-off-by: Misha Brukman <mbrukman@google.com>

mbrukman committed:
Add `latest` as a symlink to `master` dir

Signed-off-by: Misha Brukman <mbrukman@google.com>
